test/regression.py

```diff
@@ -2069,18 +2101,18 @@ def testatommasses(self):
 
     "Jaguar/Jaguar4.2/dvb_gopt.out":    JaguarGeoOptTest_nmo45,
     "Jaguar/Jaguar4.2/dvb_gopt_b.out":  GenericGeoOptTest,
-    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_sp_b.out":    JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_un_sp.out":   JaguarSPunTest_nmo_all,
+    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45_nogeo,

```

Comment:
It's unclear to me why we would expect geometry optimization for an SP job anyway. Can you try use JaguarSPTest its variant here?